vroom/setupvroom.vim

```diff
@@ -0,0 +1,41 @@
+" Copyright 2015 Google Inc. All rights reserved.
```

Comment:
Added a copyright notice here. Honestly, I hate having them in every file, but seems like it ought to be here in this file.